docs/release-notes/cloud/cloud-2023-03.md

```diff
@@ -0,0 +1,137 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud March 2023.
+included_jira_versions: ['2023.03']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.7.116
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.0.9
+---
+
+# Cloud March 2023
+
+These release notes are for the Codacy Cloud updates during March 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues without release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/PLUTO-388 Mention at least new API endpoint
+Bugs and Community Issues:
+-   https://codacy.atlassian.net/browse/TS-288 Adds ESLint plugin
+-   https://codacy.atlassian.net/browse/TS-214
+-   https://codacy.atlassian.net/browse/TS-109
+-   https://codacy.atlassian.net/browse/IO-456
```

Comment:
@prcr I set this one to not publish since it was reported internally and is not very visible to the end users (minor UI messaging update).